### PR TITLE
execute, firm, aggregate, SS

### PR DESCRIPTION
**Pull request by LeiliPR**

### CODE REVIEW

**Comment by LeiliPR** [comment]
Sorry, I just found out that SS and Firm are complete, I got conflict here so I am removing them.

**Comment by saharnazb** [comment]
SS is not complete for mine. If yours is done, let it be. I'll remove tomorrow morning before my flight and hopefully before others use it. Or we can check and merge both.

File: scripts/SS.py [hunk 1]
```diff
@@ -1,58 +0,0 @@
...
-    w_params = (alpha, A)
-
-    while dist > mindist and ss_iter < maxiter:
-        ss_iter += 1
-        K, L = KL_init
-        r = firms.get_r(K, L, r_params)
-        w = firms.get_w(K, L, w_params)
-        c1_guess = 0.5
-        c1_params = (r, w, beta, sigma, chi_n, l_tilde, b, upsilon, S)
-        results_c1 = opt.root(hh.get_b_sp1, c1_guess, params=(c1_params))
```
Comment (saharnazb) [inline scripts/SS.py]: Some of these functions are not defined in "households.py" function. I tried to check this SS with mine and household.py but have to check-in and board now. Please check them. There are some changes required to be done in the current SS and households.py
Thank you

**Review by saharnazb — CHANGES_REQUESTED** [review]

**Comment by jdebacker** [comment]
Merging in these new files from @LeiliPR

**Pull request merged**